packages/hydra-cli/src/templates/graphql-server/warthog.env.yml

```diff
@@ -1,5 +1,6 @@
 default_env: &default_env
     WARTHOG_INTROSPECTION: true
+    WARTHOG_SUBSCRIPTIONS: true
```

Comment:
do have to set this in the `.env` as well?

Reply:
It's not needed. `.env` is generated from the `yml` file with `warthog:configure`

Reply:
thanks!